Tests/MongoSwiftSyncTests/SyncChangeStreamTests.swift

```diff
@@ -369,8 +387,12 @@ final class SyncChangeStreamTests: MongoSwiftTestCase {
      * pipeline and options, except for the addition/update of a resumeToken.
      */
     func testChangeStreamAutomaticResume() throws {
-        guard MongoSwiftTestCase.topologyType != .single else {
-            print(unsupportedTopologyMessage(testName: self.name))
+        let testRequirements = TestRequirement(
+            acceptableTopologies: ["replicaSet", "sharded"].map { TestTopologyConfiguration(from: $0) })
```

Comment:
ditto enum cases directly and `)` formatting

Reply:
done